source/Cosmos.HAL/Cosmos.HAL.csproj

```diff
@@ -8,6 +8,10 @@
         <Company>Cosmos</Company>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+      <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
```

Comment:
Code in HAL should never be unsafe. The ACPI implementation should be separated between Core and HAL.